test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require "test_helper"
```

Comment:
Whats this file about?

Reply:
This file holds the default configuration for your system tests. Individual system tests inherits from the `ApplicationSystemTestCase` class.